### PR TITLE
git subrepo pull (merge) lib/astc-encoder

### PR DESCRIPTION
**Pull request by wasimabbas-arm**
subrepo:
  subdir:   "lib/astc-encoder"
  merged:   "adc637b0"
upstream:
  origin:   "https://github.com/ARM-software/astc-encoder.git"
  branch:   "main"
  commit:   "42a8f6ee"
git-subrepo:
  version:  "0.4.3"
  origin:   "https://github.com/ingydotnet/git-subrepo"
  commit:   "2f68596"

### CODE REVIEW

File: lib/astc-encoder/.gitrepo [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [subrepo]
 	remote = https://github.com/ARM-software/astc-encoder.git
 	branch = main
-	commit = e7cb1e453968b0e16e48ef6d68fc9d1227d8a378
+	commit = 42a8f6ee01715f45edffb6773e34b8bb914a47df
 	parent = b87158d371723ba9670f5dc13e50a3e2a2799eba
```
Comment (MarkCallow) [inline lib/astc-encoder/.gitrepo:10]: Change this to the correct parent of this `subrepo pull` which is 2b1c072a13e9b69ada8457665b51ed5b59ddb408.

**Review by MarkCallow — CHANGES_REQUESTED** [review]
Looks fine but the parent in the .gitrepo file must be fixed. It still shows the parent of the previous `subrepo pull`.

File: lib/astc-encoder/.gitrepo (hunk 1 above)
Comment (wasimabbas-arm) [reply lib/astc-encoder/.gitrepo]: done in https://github.com/KhronosGroup/KTX-Software/pull/537/commits/7b67ad9c03d8c8a927e25fbefef1f619c56c8be5

**Review by MarkCallow — APPROVED** [review]

**Comment by MarkCallow** [comment]
I just merged this and in doing so saw that a large number of files `astc-encoder/Test/Images//*/*_results.csv have been added. Was this intentional. Why would test results be added to the repo?

**Comment by wasimabbas-arm** [comment]
> astc-encoder/Test/Images///_results.csv

These were always there. Just upgraded due to a big change. These are reference scores, used for CI stuff. Pete has done a lot of changes recently. We could do another upgrade but its an active project. We don't really need it unless we find a major bug.

**Pull request merged**